demisto_sdk/commands/test_content/execute_test_content.py

```diff
@@ -40,7 +44,10 @@ def _add_pr_comment(comment, logging_module):
                 response = requests.get(issue_url, headers=headers, verify=False)
                 issue_comments = _handle_github_response(response, logging_module)
                 for existing_comment in issue_comments:
-                    if SKIPPED_CONTENT_COMMENT in existing_comment.get('body'):
+                    if is_skipped_tests_flow and SKIPPED_CONTENT_COMMENT in existing_comment.get('body'):
+                        comment_url = existing_comment.get('url')
+                        requests.delete(comment_url, headers=headers, verify=False)
+                    if not is_skipped_tests_flow and COVERAGE_REPORT_COMMENT in existing_comment.get('body'):
                         comment_url = existing_comment.get('url')
                         requests.delete(comment_url, headers=headers, verify=False)
```

Comment:
```suggestion
                    if (is_skipped_tests_flow and SKIPPED_CONTENT_COMMENT in existing_comment.get('body')) or (not is_skipped_tests_flow and COVERAGE_REPORT_COMMENT in existing_comment.get('body')):
                        comment_url = existing_comment.get('url')
                        requests.delete(comment_url, headers=headers, verify=False)
```

duplicate code, please verify my suggestion makes sense before applying.